### ISSUE TITLE
Apollo client ios support query batching/merging?

### ISSUE THREAD

**Issue by ArunVicky001** [apollographql/apollo-ios#3088]
### Question

I am using apollo client 1.2.0 version. I want execute multiple query in parallel. is it possible in this version? 

example 

```
import Apollo

let apolloClient = ApolloClient(url: URL(string: "https://api.graphql.org/"))

let query1 = """
query {
  viewer {
    name
  }
}
"""

let query2 = """
query {
  repository(owner: "apollographql", name: "apollo-client") {
    name
  }
}
"""

let batchQuery = BatchQuery([query1, query2])

apolloClient.send(batchQuery) { result in
  switch result {
  case .success(let data):
    print(data.data?.viewer?.name)
    print(data.data?.repository?.name)
  case .failure(let error):
    print(error)
  }
}
```

**Comment by calvincestari** [comment]
@ArunVicky001 please don't open a new issue for the [same request](https://github.com/apollographql/apollo-ios/issues/192#issuecomment-1602620043). I'll respond in the older issue.
